PolicyDecisionPoint/src/main/java/it/cnr/iit/ucs/pdp/PolicyDecisionPoint.java

```diff
@@ -286,13 +280,13 @@ private ResponseCtx getResponseCtx( int result, String code, String message ) {
         return new ResponseCtx( new Result( result, status ) );
     }
 
-    private void buildJournal( String journalDir ) {
-        try {
-            File file = new File( journalDir ); // TODO UCS-33 NOSONAR
-            FileUtility.createPathIfNotExists( file );
-            journal = JournalBuilder.of( file ).open();
-        } catch( Exception e ) {
-            throw new IllegalArgumentException( e.getMessage() );
+    private void write( String policy, String request, String response ) {
+        if( !journalInterface.isPresent() ) {
+            return;
+        } else {
```

Comment:
you do not really need an else here if you do an early return when not present

Reply:
as before, I've noticed it after the pull request was done